bin/secret-service/src/seeded_impl/wallet.rs

```diff
@@ -0,0 +1,66 @@
+//! In-memory persistence for operator's secret data.
```

Comment:
This needs to be `Zeroizable`, in `strata-primitives`, we have `ZeroizableXpriv` and `ZeroizableKeypair`.
Either we add these or we open a ticket for the future, your choice.